GVFS/GVFS.NativeHooks.Common/common.mac.cpp

```diff
@@ -21,7 +21,7 @@ PATH_STRING GetGVFSPipeName(const char *appName)
 {
     // The pipe name is built using the path of the GVFS enlistment root.
     // Start in the current directory and walk up the directory tree
-    // until we find a folder that contains the ".gvfs" folder
+    // until we find a folder that contains the ".vfsforgit" folder
```

Comment:
Hard stop. If you are changing this on mac, then you will break our pilot customers.